AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts

```diff
@@ -91,7 +101,7 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
         this.populateFormFields(existingJudge);
 
         this.form = this.fb.group({
-            judgeName: [this.judge.display_name, Validators.required],
+            judgeName: [this.judge.email, Validators.required],
```

Comment:
is calling this judgeName still correct now it contains the email?

Reply:
This is the contact email, the second one